styx-cli/src/main/java/com/spotify/styx/cli/CliMain.java

```diff
@@ -771,7 +848,9 @@ public Subparser parser(Subparsers subCommands) {
   private enum WorkflowCommand {
     SHOW("get", "Show info about a specific workflow"),
     CREATE("", "Create or update a workflow"),
-    DELETE("", "Delete a workflow");
+    DELETE("", "Delete a workflow"),
```

Comment:
since these take multiple workflows, maybe it should say `Delete workflow(s)` and so on

Reply:
Good points and will fix.